include/pfnet/json_macros.h

```diff
@@ -13,6 +13,9 @@
 
 #include <string.h>
 
+// constants
+#define JSON_STR_BUFFER_EXTRA 10
```

Comment:
Is this still needed? Can we just use the "json" buffers which are larger than the "regular" buffers?